### PR TITLE
Removes radium from the Drawer component

### PR DESCRIPTION
**Pull request by TayteStokes**
This change removes using radium to apply styles to the Drawer component and implements using glamor to apply those styles instead.

// @garrowp @DrewWarrenTIY @jacob-pettingill @phantomxc

### CODE REVIEW

File: src/components/Drawer.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const keycode = require('keycode');
 const PropTypes = require('prop-types');
 const React = require('react');
 const Velocity = require('velocity-animate');
-const { StyleRoot } = require('radium');
+import { css } from 'glamor';
```
Comment (jacob-pettingill) [inline src/components/Drawer.js:11]: Was @whiteinge trying to remove glamor from the OS components? Where did we get on that? Is there an package we are trying to get to?

**Comment by jacob-pettingill** [comment]
Other than the Glamor question, this looks good to me! Do we have a screenshot or vid showing this working?

File: src/components/Drawer.js [hunk 2]
```diff
@@ -227,7 +227,7 @@ class Drawer extends React.Component {
     const titleUniqueId = _uniqueId('mx-drawer-title-');
 
     return (
-      <StyleRoot>
+      <div>
```
Comment (phantomxc) [inline src/components/Drawer.js:230]: Do we still need this div?

File: src/components/Drawer.js (hunk 1 above)
Comment (jgerlach) [reply src/components/Drawer.js:11]: I had the same question.

Comment (TayteStokes) [reply src/components/Drawer.js:11]: I think that is the plan, but it hasn't been implemented yet.

File: src/components/Drawer.js (hunk 2 above)
Comment (TayteStokes) [reply src/components/Drawer.js]: Nope, I should have removed that. Just pushed that change.

**Comment by TayteStokes** [comment]
@jacob-pettingill I provided screen recordings in the Gitlab issue, but here they are as well.

Before:

before

After:

after

**Review by phantomxc — APPROVED** [review]

File: src/components/Drawer.js (hunk 1 above)
Comment (whiteinge) [reply src/components/Drawer.js:11]: PR #900 makes the switch off Glamor. Looks like there's a new npm audit failure :roll_eyes: so I'll get that PR updated and then I think it's ready for prime-time.

That said, this PR looks fine and I don't see any reason the Glamor switch should hold this one up.

File: src/components/Drawer.js [hunk 3]
```diff
@@ -240,12 +240,15 @@ class Drawer extends React.Component {
             <div
               aria-describedby={this.props['aria-describedby']}
               aria-labelledby={this.props['aria-labelledby'] || titleUniqueId}
+              className={css({ ...styles.component, ...this.props.style })}
```
[the remark this answers is not in the document]
Comment (TayteStokes) [reply src/components/Drawer.js:242]: `style` is actually the name of the prop that we are using. If you take a look at the `FlexDrawer` component in Raja, you can see an example of how it's being used. Do you think this should be updated to `styles`? We would have to update everywhere that is passing styles through the style prop as well.

[the remark this answers is not in the document]
Comment (DrewWarrenTIY) [reply src/components/Drawer.js:242]: we talked about this. It probably "should" be styles with an s, but is currently being passed the object in props as style without an s so we should leave it to minimize chances of anything breaking

**Pull request merged**